### ISSUE TITLE
intern-cli exits with 0 after test failure

### ISSUE THREAD

**Issue by sholladay** [theintern/intern#668]
This prints foo:

```sh
intern run && echo 'foo'
```

This does not:

```sh
intern-client config=tests/intern && echo 'foo'
```

```sh
intern --version
  
  intern-cli: 0.1.0
  intern: 3.3.1
```

### Reproduce

```sh
mkdir repro && cd repro &&
npm init -y && npm install --save-dev intern &&
intern init
```
Now modify the assertion in `tests/unit/hello.js` to fail:

`assert.strictEqual(foo, 'hello');` => `assert.strictEqual(foo, 'helloz');`

`intern` will display the failure, but exit with code 0, thus this will print foo.

```sh
intern run && echo 'foo'
```

Expected behavior is a non-zero exit status when a test fails.

**Comment by jason0x43** [comment]
This has been fixed in [intern-cli 0.1.1](https://github.com/theintern/intern-cli/releases/tag/0.1.1).

**Comment by sholladay** [comment]
That was fast. Thank you!
